tests/test_tasks/test_task_functions.py

```diff
@@ -129,6 +129,24 @@ def test_get_task(self):
             self.workdir, 'org', 'openml', 'test', "datasets", "1", "dataset.arff"
         )))
 
+    def test_get_task_lazy(self):
+        task = openml.tasks.get_task(2, download_data=False)
+        self.assertIsInstance(task, OpenMLTask)
```

Comment:
Could you please add a check that the class labels are retrieved despite the data not downloaded?